src/popup/router/components/NodeConnectionStatus.vue

```diff
@@ -63,4 +63,12 @@ export default {
 .connect-node {
   background: variables.$color-bg-3;
 }
+
+@media screen and (max-height: 250px) {
```

Comment:
Maybe let's add this to `mixins.scss` in case we will need more design changes for collapsed wallet